### PR TITLE
[plugin.video.milbtv] 2021.05.15

### PR DESCRIPTION
**Pull request by tonywagner**
### Description

- General
  - Add-on name: MiLB.TV®
  - Add-on ID: plugin.video.milbtv
  - Version number: 2021.05.15
  - Kodi/repository version: matrix
- Code location
  - URL: https://github.com/tonywagner/plugin.video.milbtv
 
Watch live and archived minor league baseball games with your MiLB.TV account

New add-on, initial pull request

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

**Comment by tonywagner** [comment]
Hello @enen92 and @basrieter -- is there anything further I need to do to start the addon checker here? Thanks!

File: plugin.video.milbtv/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Comment (basrieter) [inline plugin.video.milbtv/.gitignore]: Exclude this file

**Review by basrieter — CHANGES_REQUESTED** [review]
Please remove the .gitignore

I will check the rest later

File: plugin.video.milbtv/.gitignore (hunk 1 above)
Comment (tonywagner) [reply plugin.video.milbtv/.gitignore]: Done!

**Comment by basrieter** [comment]
How does this add-on relate to https://github.com/eracknaphobia/plugin.video.mlbtv? It almost seems like a copy?

**Comment by tonywagner** [comment]
> How does this add-on relate to https://github.com/eracknaphobia/plugin.video.mlbtv? It almost seems like a copy?

It is based on that add-on. That is the version for major league baseball, but this is for minor league baseball (a separate streaming service).

**Comment by basrieter** [comment]
> > How does this add-on relate to https://github.com/eracknaphobia/plugin.video.mlbtv? It almost seems like a copy?
> 
> It is based on that add-on. That is the version for major league baseball, but this is for minor league baseball (a separate streaming service).

Ok, clear. In that case you will need to comply with the GNU General Public License v2.0 conditions:

- [x] License and copyright notice
- [x] State changes
- [x] Disclose source
- [x] Same license

To comply with the first one you basically should add an SPDX line to all python files: `SPDX-License-Identifier: GPL-2.0-or-later` and add the original copyright notice of `plugin.video.mlbtv` somewhere (officially in all the files). For the second one you will need to state your changes you made to the copyrighted material.

**Comment by tonywagner** [comment]
> To comply with the first one you basically should add an SPDX line to all python files: `SPDX-License-Identifier: GPL-2.0-or-later` and add the original copyright notice of `plugin.video.mlbtv` somewhere (officially in all the files). For the second one you will need to state your changes you made to the copyrighted material.

Thanks! Would something like this be sufficient, at the top of all of the modified Python code files?

```
# SPDX-License-Identifier: GPL-2.0-or-later
# Original plugin.video.mlbtv © eracknaphobia
# Modified for MiLB.TV compatibility and code cleanup
```

**Comment by basrieter** [comment]
Yes, that would suffice I guess.

**Review by basrieter — APPROVED** [review]

**Pull request merged**